web/main/models.py

```diff
@@ -1653,6 +1657,22 @@ def export_content(self, request):
             return rich_text_export(self.resource.content, request=request, id_prefix=str(self.id))
         return self.resource.content
 
+    def get_ali_license_text(self):
+        with open(f"{settings.STATIC_ROOT}/data/ali_materials.json", "r") as file:
```

Comment:
Loading this file every time will be slow -- I think you could load this once in settings_base.py and store the dict as settings.ALI_MATERIALS or somesuch.